packages/a11y-testing/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "lib",
+    "target": "es6",
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strictNullChecks": true,
+    "noImplicitAny": true,
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+    "preserveConstEnums": true,
+    "lib": ["es2017", "dom"],
+    "types": ["jest"]
```

Comment:
```suggestion
    "types": [""]
```

Reply:
not taking the suggestion as I use it in "validate.ts" file, this part of the code:
` test(rule.stringify(), () => {
          validateSlot(rule, testFacade);
        });`